var/spack/repos/builtin/packages/fermikit/package.py

```diff
@@ -17,6 +17,12 @@ class Fermikit(MakefilePackage):
             submodules=True)
 
     depends_on('zlib')
+    depends_on('sse2neon', when='target=aarch64:')
+
+    patch('bwa-ksw_for_aarch64.patch', when='target=aarch64:')
+    patch('fermi2-ksw_for_aarch64.patch', when='target=aarch64:')
+    patch('seqtk-ksw_for_aarch.patch', when='target=aarch64:')
+    patch('trimadap-ksw_for_aarch.patch', when='target=aarch64:')
```

Comment:
Can we have a unique patch for the four file:
```python
patch('sse2neon.patch', when='target=aarch64:')
```
?

Reply:
Merge into one patch file.